pkgs/development/python-modules/asyncstdlib-fw/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  pdm-backend,
+}:
+
+buildPythonPackage rec {
+  pname = "asyncstdlib-fw";
+  version = "3.13.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "asyncstdlib_fw";
+    inherit version;
+    hash = "sha256-Ua0JTCBMWTbDBA84wy/W1UmzkcmA8h8foJW2X7aAah8=";
+  };
+
+  build-system = [
+    pdm-backend
+  ];
+
+  doCheck = false; # no tests supplied
+
+  pythonImportsCheck = [
+    "asyncstdlib"
+  ];
+
+  meta = {
+    description = "Fork of asyncstdlib that work with fireworks-ai";
+    homepage = "https://pypi.org/project/asyncstdlib-fw/";
```

Comment:
Why not use the GH repo as the homepage?

Reply:
There is no GH repo for this (yet). The developers aren't opening up the repo until this is stable and changing more slowly.